api/indexer.oas2.json

```diff
@@ -853,6 +853,67 @@
         }
       }
     },
+    "/v2/blocks": {
+      "get": {
+        "description": "Search for blocks. Blocks are returned in ascending round order. Transactions are not included in the output.",
```

Comment:
Since there are no transactions, the endpoint might better be called `blockheaders`. I think that's what the similar algod endpoint under development is called.

Reply:
The route was changed to `GET /v2/block-headers`, in https://github.com/algorand/indexer/pull/1638/commits/a302e5b9948fa1dace8f77f2b31bfeb8227d0bac